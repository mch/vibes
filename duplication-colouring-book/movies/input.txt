file 'Duplication/Slide1.jpg'
duration 4.0
file 'Duplication/Slide2.jpg'
duration 1.491937
file 'Duplication/Slide3.jpg'
duration 3.954840
file 'Duplication/Slide4.jpg'
duration 1.475808
file 'Duplication/Slide5.jpg'
duration 3.909679
file 'Duplication/Slide6.jpg'
duration 1.459678
file 'Duplication/Slide7.jpg'
duration 3.864519
file 'Duplication/Slide8.jpg'
duration 1.443549
file 'Duplication/Slide9.jpg'
duration 3.819358
file 'Duplication/Slide10.jpg'
duration 1.427420
file 'Duplication/Slide11.jpg'
duration 3.774194
file 'Duplication/Slide12.jpg'
duration 1.411292
file 'Duplication/Slide13.jpg'
duration 3.729034
file 'Duplication/Slide14.jpg'
duration 1.395163
file 'Duplication/Slide15.jpg'
duration 3.683873
file 'Duplication/Slide16.jpg'
duration 1.379033
file 'Duplication/Slide17.jpg'
duration 3.638713
file 'Duplication/Slide18.jpg'
duration 1.362904
file 'Duplication/Slide19.jpg'
duration 3.593549
file 'Duplication/Slide20.jpg'
duration 1.346775
file 'Duplication/Slide21.jpg'
duration 3.548388
file 'Duplication/Slide22.jpg'
duration 1.330647
file 'Duplication/Slide23.jpg'
duration 3.503228
file 'Duplication/Slide24.jpg'
duration 1.314517
file 'Duplication/Slide25.jpg'
duration 3.458067
file 'Duplication/Slide26.jpg'
duration 1.298388
file 'Duplication/Slide27.jpg'
duration 3.412907
file 'Duplication/Slide28.jpg'
duration 1.282259
file 'Duplication/Slide29.jpg'
duration 3.367743
file 'Duplication/Slide30.jpg'
duration 1.266130
file 'Duplication/Slide31.jpg'
duration 3.322582
file 'Duplication/Slide32.jpg'
duration 1.250000
file 'Duplication/Slide33.jpg'
duration 3.277422
file 'Duplication/Slide34.jpg'
duration 1.233872
file 'Duplication/Slide35.jpg'
duration 3.232261
file 'Duplication/Slide36.jpg'
duration 1.217743
file 'Duplication/Slide37.jpg'
duration 3.187097
file 'Duplication/Slide38.jpg'
duration 1.201614
file 'Duplication/Slide39.jpg'
duration 3.141937
file 'Duplication/Slide40.jpg'
duration 1.185485
file 'Duplication/Slide41.jpg'
duration 3.096776
file 'Duplication/Slide42.jpg'
duration 1.169355
file 'Duplication/Slide43.jpg'
duration 3.051616
file 'Duplication/Slide44.jpg'
duration 1.153227
file 'Duplication/Slide45.jpg'
duration 3.006455
file 'Duplication/Slide46.jpg'
duration 1.137098
file 'Duplication/Slide47.jpg'
duration 2.961291
file 'Duplication/Slide48.jpg'
duration 1.120969
file 'Duplication/Slide49.jpg'
duration 2.916131
file 'Duplication/Slide50.jpg'
duration 1.104839
file 'Duplication/Slide51.jpg'
duration 2.870970
file 'Duplication/Slide52.jpg'
duration 1.088710
file 'Duplication/Slide53.jpg'
duration 2.825810
file 'Duplication/Slide54.jpg'
duration 1.072582
file 'Duplication/Slide55.jpg'
duration 2.780646
file 'Duplication/Slide56.jpg'
duration 1.056453
file 'Duplication/Slide57.jpg'
duration 2.735485
file 'Duplication/Slide58.jpg'
duration 1.040324
file 'Duplication/Slide59.jpg'
duration 2.690325
file 'Duplication/Slide60.jpg'
duration 1.024194
file 'Duplication/Slide61.jpg'
duration 2.645164
file 'Duplication/Slide62.jpg'
duration 1.008065
file 'Duplication/Slide63.jpg'
duration 2.600000
file 'Duplication/Slide64.jpg'
duration 0.991937
file 'Duplication/Slide65.jpg'
duration 2.554840
file 'Duplication/Slide66.jpg'
duration 0.975808
file 'Duplication/Slide67.jpg'
duration 2.509679
file 'Duplication/Slide68.jpg'
duration 0.959678
file 'Duplication/Slide69.jpg'
duration 2.464519
file 'Duplication/Slide70.jpg'
duration 0.943549
file 'Duplication/Slide71.jpg'
duration 2.419358
file 'Duplication/Slide72.jpg'
duration 0.927420
file 'Duplication/Slide73.jpg'
duration 2.374194
file 'Duplication/Slide74.jpg'
duration 0.911292
file 'Duplication/Slide75.jpg'
duration 2.329034
file 'Duplication/Slide76.jpg'
duration 0.895163
file 'Duplication/Slide77.jpg'
duration 2.283873
file 'Duplication/Slide78.jpg'
duration 0.879033
file 'Duplication/Slide79.jpg'
duration 2.238713
file 'Duplication/Slide80.jpg'
duration 0.862904
file 'Duplication/Slide81.jpg'
duration 2.193549
file 'Duplication/Slide82.jpg'
duration 0.846775
file 'Duplication/Slide83.jpg'
duration 2.148388
file 'Duplication/Slide84.jpg'
duration 0.830647
file 'Duplication/Slide85.jpg'
duration 2.103228
file 'Duplication/Slide86.jpg'
duration 0.814517
file 'Duplication/Slide87.jpg'
duration 2.058067
file 'Duplication/Slide88.jpg'
duration 0.798388
file 'Duplication/Slide89.jpg'
duration 2.012907
file 'Duplication/Slide90.jpg'
duration 0.782259
file 'Duplication/Slide91.jpg'
duration 1.967743
file 'Duplication/Slide92.jpg'
duration 0.766130
file 'Duplication/Slide93.jpg'
duration 1.922582
file 'Duplication/Slide94.jpg'
duration 0.750000
file 'Duplication/Slide95.jpg'
duration 1.877422
file 'Duplication/Slide96.jpg'
duration 0.733872
file 'Duplication/Slide97.jpg'
duration 1.832261
file 'Duplication/Slide98.jpg'
duration 0.717743
file 'Duplication/Slide99.jpg'
duration 1.787097
file 'Duplication/Slide100.jpg'
duration 0.701614
file 'Duplication/Slide101.jpg'
duration 1.741937
file 'Duplication/Slide102.jpg'
duration 0.685485
file 'Duplication/Slide103.jpg'
duration 1.696776
file 'Duplication/Slide104.jpg'
duration 0.669355
file 'Duplication/Slide105.jpg'
duration 1.651616
file 'Duplication/Slide106.jpg'
duration 0.653227
file 'Duplication/Slide107.jpg'
duration 1.606455
file 'Duplication/Slide108.jpg'
duration 0.637098
file 'Duplication/Slide109.jpg'
duration 1.561291
file 'Duplication/Slide110.jpg'
duration 0.620969
file 'Duplication/Slide111.jpg'
duration 1.516131
file 'Duplication/Slide112.jpg'
duration 0.604839
file 'Duplication/Slide113.jpg'
duration 1.470970
file 'Duplication/Slide114.jpg'
duration 0.588710
file 'Duplication/Slide115.jpg'
duration 1.425810
file 'Duplication/Slide116.jpg'
duration 0.572582
file 'Duplication/Slide117.jpg'
duration 1.380646
file 'Duplication/Slide118.jpg'
duration 0.556453
file 'Duplication/Slide119.jpg'
duration 1.335485
file 'Duplication/Slide120.jpg'
duration 0.540324
file 'Duplication/Slide121.jpg'
duration 1.290325
file 'Duplication/Slide122.jpg'
duration 0.524194
file 'Duplication/Slide123.jpg'
duration 1.245164
file 'Duplication/Slide124.jpg'
duration 0.508065
file 'Duplication/Slide125.jpg'
duration 1.200000
file 'Duplication/Slide126.jpg'
duration 0.491937
file 'Duplication/Slide127.jpg'
duration 1.154840
file 'Duplication/Slide128.jpg'
duration 0.475808
file 'Duplication/Slide129.jpg'
duration 1.109679
file 'Duplication/Slide130.jpg'
duration 0.459678
file 'Duplication/Slide131.jpg'
duration 1.064519
file 'Duplication/Slide132.jpg'
duration 0.443549
file 'Duplication/Slide133.jpg'
duration 1.019358
file 'Duplication/Slide134.jpg'
duration 0.427420
file 'Duplication/Slide135.jpg'
duration 0.974194
file 'Duplication/Slide136.jpg'
duration 0.411292
file 'Duplication/Slide137.jpg'
duration 0.929034
file 'Duplication/Slide138.jpg'
duration 0.395163
file 'Duplication/Slide139.jpg'
duration 0.883873
file 'Duplication/Slide140.jpg'
duration 0.379033
file 'Duplication/Slide141.jpg'
duration 0.838713
file 'Duplication/Slide142.jpg'
duration 0.362904
file 'Duplication/Slide143.jpg'
duration 0.793549
file 'Duplication/Slide144.jpg'
duration 0.346775
file 'Duplication/Slide145.jpg'
duration 0.748388
file 'Duplication/Slide146.jpg'
duration 0.330647
file 'Duplication/Slide147.jpg'
duration 0.703228
file 'Duplication/Slide148.jpg'
duration 0.314517
file 'Duplication/Slide149.jpg'
duration 0.658067
file 'Duplication/Slide150.jpg'
duration 0.298388
file 'Duplication/Slide151.jpg'
duration 0.612907
file 'Duplication/Slide152.jpg'
duration 0.282259
file 'Duplication/Slide153.jpg'
duration 0.567743
file 'Duplication/Slide154.jpg'
duration 0.266130
file 'Duplication/Slide155.jpg'
duration 0.522582
file 'Duplication/Slide156.jpg'
duration 0.250000
file 'Duplication/Slide156.jpg'
