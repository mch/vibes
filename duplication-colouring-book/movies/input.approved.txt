file '/Users/mch/Downloads/Duplication/Slide1.jpg'
duration 4.0
file '/Users/mch/Downloads/Duplication/Slide2.jpg'
duration 1.491937
file '/Users/mch/Downloads/Duplication/Slide3.jpg'
duration 3.954840
file '/Users/mch/Downloads/Duplication/Slide4.jpg'
duration 1.475808
file '/Users/mch/Downloads/Duplication/Slide5.jpg'
duration 3.909679
file '/Users/mch/Downloads/Duplication/Slide6.jpg'
duration 1.459678
file '/Users/mch/Downloads/Duplication/Slide7.jpg'
duration 3.864519
file '/Users/mch/Downloads/Duplication/Slide8.jpg'
duration 1.443549
file '/Users/mch/Downloads/Duplication/Slide9.jpg'
duration 3.819358
file '/Users/mch/Downloads/Duplication/Slide10.jpg'
duration 1.427420
file '/Users/mch/Downloads/Duplication/Slide11.jpg'
duration 3.774194
file '/Users/mch/Downloads/Duplication/Slide12.jpg'
duration 1.411292
file '/Users/mch/Downloads/Duplication/Slide13.jpg'
duration 3.729034
file '/Users/mch/Downloads/Duplication/Slide14.jpg'
duration 1.395163
file '/Users/mch/Downloads/Duplication/Slide15.jpg'
duration 3.683873
file '/Users/mch/Downloads/Duplication/Slide16.jpg'
duration 1.379033
file '/Users/mch/Downloads/Duplication/Slide17.jpg'
duration 3.638713
file '/Users/mch/Downloads/Duplication/Slide18.jpg'
duration 1.362904
file '/Users/mch/Downloads/Duplication/Slide19.jpg'
duration 3.593549
file '/Users/mch/Downloads/Duplication/Slide20.jpg'
duration 1.346775
file '/Users/mch/Downloads/Duplication/Slide21.jpg'
duration 3.548388
file '/Users/mch/Downloads/Duplication/Slide22.jpg'
duration 1.330647
file '/Users/mch/Downloads/Duplication/Slide23.jpg'
duration 3.503228
file '/Users/mch/Downloads/Duplication/Slide24.jpg'
duration 1.314517
file '/Users/mch/Downloads/Duplication/Slide25.jpg'
duration 3.458067
file '/Users/mch/Downloads/Duplication/Slide26.jpg'
duration 1.298388
file '/Users/mch/Downloads/Duplication/Slide27.jpg'
duration 3.412907
file '/Users/mch/Downloads/Duplication/Slide28.jpg'
duration 1.282259
file '/Users/mch/Downloads/Duplication/Slide29.jpg'
duration 3.367743
file '/Users/mch/Downloads/Duplication/Slide30.jpg'
duration 1.266130
file '/Users/mch/Downloads/Duplication/Slide31.jpg'
duration 3.322582
file '/Users/mch/Downloads/Duplication/Slide32.jpg'
duration 1.250000
file '/Users/mch/Downloads/Duplication/Slide33.jpg'
duration 3.277422
file '/Users/mch/Downloads/Duplication/Slide34.jpg'
duration 1.233872
file '/Users/mch/Downloads/Duplication/Slide35.jpg'
duration 3.232261
file '/Users/mch/Downloads/Duplication/Slide36.jpg'
duration 1.217743
file '/Users/mch/Downloads/Duplication/Slide37.jpg'
duration 3.187097
file '/Users/mch/Downloads/Duplication/Slide38.jpg'
duration 1.201614
file '/Users/mch/Downloads/Duplication/Slide39.jpg'
duration 3.141937
file '/Users/mch/Downloads/Duplication/Slide40.jpg'
duration 1.185485
file '/Users/mch/Downloads/Duplication/Slide41.jpg'
duration 3.096776
file '/Users/mch/Downloads/Duplication/Slide42.jpg'
duration 1.169355
file '/Users/mch/Downloads/Duplication/Slide43.jpg'
duration 3.051616
file '/Users/mch/Downloads/Duplication/Slide44.jpg'
duration 1.153227
file '/Users/mch/Downloads/Duplication/Slide45.jpg'
duration 3.006455
file '/Users/mch/Downloads/Duplication/Slide46.jpg'
duration 1.137098
file '/Users/mch/Downloads/Duplication/Slide47.jpg'
duration 2.961291
file '/Users/mch/Downloads/Duplication/Slide48.jpg'
duration 1.120969
file '/Users/mch/Downloads/Duplication/Slide49.jpg'
duration 2.916131
file '/Users/mch/Downloads/Duplication/Slide50.jpg'
duration 1.104839
file '/Users/mch/Downloads/Duplication/Slide51.jpg'
duration 2.870970
file '/Users/mch/Downloads/Duplication/Slide52.jpg'
duration 1.088710
file '/Users/mch/Downloads/Duplication/Slide53.jpg'
duration 2.825810
file '/Users/mch/Downloads/Duplication/Slide54.jpg'
duration 1.072582
file '/Users/mch/Downloads/Duplication/Slide55.jpg'
duration 2.780646
file '/Users/mch/Downloads/Duplication/Slide56.jpg'
duration 1.056453
file '/Users/mch/Downloads/Duplication/Slide57.jpg'
duration 2.735485
file '/Users/mch/Downloads/Duplication/Slide58.jpg'
duration 1.040324
file '/Users/mch/Downloads/Duplication/Slide59.jpg'
duration 2.690325
file '/Users/mch/Downloads/Duplication/Slide60.jpg'
duration 1.024194
file '/Users/mch/Downloads/Duplication/Slide61.jpg'
duration 2.645164
file '/Users/mch/Downloads/Duplication/Slide62.jpg'
duration 1.008065
file '/Users/mch/Downloads/Duplication/Slide63.jpg'
duration 2.600000
file '/Users/mch/Downloads/Duplication/Slide64.jpg'
duration 0.991937
file '/Users/mch/Downloads/Duplication/Slide65.jpg'
duration 2.554840
file '/Users/mch/Downloads/Duplication/Slide66.jpg'
duration 0.975808
file '/Users/mch/Downloads/Duplication/Slide67.jpg'
duration 2.509679
file '/Users/mch/Downloads/Duplication/Slide68.jpg'
duration 0.959678
file '/Users/mch/Downloads/Duplication/Slide69.jpg'
duration 2.464519
file '/Users/mch/Downloads/Duplication/Slide70.jpg'
duration 0.943549
file '/Users/mch/Downloads/Duplication/Slide71.jpg'
duration 2.419358
file '/Users/mch/Downloads/Duplication/Slide72.jpg'
duration 0.927420
file '/Users/mch/Downloads/Duplication/Slide73.jpg'
duration 2.374194
file '/Users/mch/Downloads/Duplication/Slide74.jpg'
duration 0.911292
file '/Users/mch/Downloads/Duplication/Slide75.jpg'
duration 2.329034
file '/Users/mch/Downloads/Duplication/Slide76.jpg'
duration 0.895163
file '/Users/mch/Downloads/Duplication/Slide77.jpg'
duration 2.283873
file '/Users/mch/Downloads/Duplication/Slide78.jpg'
duration 0.879033
file '/Users/mch/Downloads/Duplication/Slide79.jpg'
duration 2.238713
file '/Users/mch/Downloads/Duplication/Slide80.jpg'
duration 0.862904
file '/Users/mch/Downloads/Duplication/Slide81.jpg'
duration 2.193549
file '/Users/mch/Downloads/Duplication/Slide82.jpg'
duration 0.846775
file '/Users/mch/Downloads/Duplication/Slide83.jpg'
duration 2.148388
file '/Users/mch/Downloads/Duplication/Slide84.jpg'
duration 0.830647
file '/Users/mch/Downloads/Duplication/Slide85.jpg'
duration 2.103228
file '/Users/mch/Downloads/Duplication/Slide86.jpg'
duration 0.814517
file '/Users/mch/Downloads/Duplication/Slide87.jpg'
duration 2.058067
file '/Users/mch/Downloads/Duplication/Slide88.jpg'
duration 0.798388
file '/Users/mch/Downloads/Duplication/Slide89.jpg'
duration 2.012907
file '/Users/mch/Downloads/Duplication/Slide90.jpg'
duration 0.782259
file '/Users/mch/Downloads/Duplication/Slide91.jpg'
duration 1.967743
file '/Users/mch/Downloads/Duplication/Slide92.jpg'
duration 0.766130
file '/Users/mch/Downloads/Duplication/Slide93.jpg'
duration 1.922582
file '/Users/mch/Downloads/Duplication/Slide94.jpg'
duration 0.750000
file '/Users/mch/Downloads/Duplication/Slide95.jpg'
duration 1.877422
file '/Users/mch/Downloads/Duplication/Slide96.jpg'
duration 0.733872
file '/Users/mch/Downloads/Duplication/Slide97.jpg'
duration 1.832261
file '/Users/mch/Downloads/Duplication/Slide98.jpg'
duration 0.717743
file '/Users/mch/Downloads/Duplication/Slide99.jpg'
duration 1.787097
file '/Users/mch/Downloads/Duplication/Slide100.jpg'
duration 0.701614
file '/Users/mch/Downloads/Duplication/Slide101.jpg'
duration 1.741937
file '/Users/mch/Downloads/Duplication/Slide102.jpg'
duration 0.685485
file '/Users/mch/Downloads/Duplication/Slide103.jpg'
duration 1.696776
file '/Users/mch/Downloads/Duplication/Slide104.jpg'
duration 0.669355
file '/Users/mch/Downloads/Duplication/Slide105.jpg'
duration 1.651616
file '/Users/mch/Downloads/Duplication/Slide106.jpg'
duration 0.653227
file '/Users/mch/Downloads/Duplication/Slide107.jpg'
duration 1.606455
file '/Users/mch/Downloads/Duplication/Slide108.jpg'
duration 0.637098
file '/Users/mch/Downloads/Duplication/Slide109.jpg'
duration 1.561291
file '/Users/mch/Downloads/Duplication/Slide110.jpg'
duration 0.620969
file '/Users/mch/Downloads/Duplication/Slide111.jpg'
duration 1.516131
file '/Users/mch/Downloads/Duplication/Slide112.jpg'
duration 0.604839
file '/Users/mch/Downloads/Duplication/Slide113.jpg'
duration 1.470970
file '/Users/mch/Downloads/Duplication/Slide114.jpg'
duration 0.588710
file '/Users/mch/Downloads/Duplication/Slide115.jpg'
duration 1.425810
file '/Users/mch/Downloads/Duplication/Slide116.jpg'
duration 0.572582
file '/Users/mch/Downloads/Duplication/Slide117.jpg'
duration 1.380646
file '/Users/mch/Downloads/Duplication/Slide118.jpg'
duration 0.556453
file '/Users/mch/Downloads/Duplication/Slide119.jpg'
duration 1.335485
file '/Users/mch/Downloads/Duplication/Slide120.jpg'
duration 0.540324
file '/Users/mch/Downloads/Duplication/Slide121.jpg'
duration 1.290325
file '/Users/mch/Downloads/Duplication/Slide122.jpg'
duration 0.524194
file '/Users/mch/Downloads/Duplication/Slide123.jpg'
duration 1.245164
file '/Users/mch/Downloads/Duplication/Slide124.jpg'
duration 0.508065
file '/Users/mch/Downloads/Duplication/Slide125.jpg'
duration 1.200000
file '/Users/mch/Downloads/Duplication/Slide126.jpg'
duration 0.491937
file '/Users/mch/Downloads/Duplication/Slide127.jpg'
duration 1.154840
file '/Users/mch/Downloads/Duplication/Slide128.jpg'
duration 0.475808
file '/Users/mch/Downloads/Duplication/Slide129.jpg'
duration 1.109679
file '/Users/mch/Downloads/Duplication/Slide130.jpg'
duration 0.459678
file '/Users/mch/Downloads/Duplication/Slide131.jpg'
duration 1.064519
file '/Users/mch/Downloads/Duplication/Slide132.jpg'
duration 0.443549
file '/Users/mch/Downloads/Duplication/Slide133.jpg'
duration 1.019358
file '/Users/mch/Downloads/Duplication/Slide134.jpg'
duration 0.427420
file '/Users/mch/Downloads/Duplication/Slide135.jpg'
duration 0.974194
file '/Users/mch/Downloads/Duplication/Slide136.jpg'
duration 0.411292
file '/Users/mch/Downloads/Duplication/Slide137.jpg'
duration 0.929034
file '/Users/mch/Downloads/Duplication/Slide138.jpg'
duration 0.395163
file '/Users/mch/Downloads/Duplication/Slide139.jpg'
duration 0.883873
file '/Users/mch/Downloads/Duplication/Slide140.jpg'
duration 0.379033
file '/Users/mch/Downloads/Duplication/Slide141.jpg'
duration 0.838713
file '/Users/mch/Downloads/Duplication/Slide142.jpg'
duration 0.362904
file '/Users/mch/Downloads/Duplication/Slide143.jpg'
duration 0.793549
file '/Users/mch/Downloads/Duplication/Slide144.jpg'
duration 0.346775
file '/Users/mch/Downloads/Duplication/Slide145.jpg'
duration 0.748388
file '/Users/mch/Downloads/Duplication/Slide146.jpg'
duration 0.330647
file '/Users/mch/Downloads/Duplication/Slide147.jpg'
duration 0.703228
file '/Users/mch/Downloads/Duplication/Slide148.jpg'
duration 0.314517
file '/Users/mch/Downloads/Duplication/Slide149.jpg'
duration 0.658067
file '/Users/mch/Downloads/Duplication/Slide150.jpg'
duration 0.298388
file '/Users/mch/Downloads/Duplication/Slide151.jpg'
duration 0.612907
file '/Users/mch/Downloads/Duplication/Slide152.jpg'
duration 0.282259
file '/Users/mch/Downloads/Duplication/Slide153.jpg'
duration 0.567743
file '/Users/mch/Downloads/Duplication/Slide154.jpg'
duration 0.266130
file '/Users/mch/Downloads/Duplication/Slide155.jpg'
duration 0.522582
file '/Users/mch/Downloads/Duplication/Slide156.jpg'
duration 0.250000
file '/Users/mch/Downloads/Duplication/Slide156.jpg'
